README.md

```diff
@@ -58,7 +58,7 @@ This is just the beginning of MicroCloud. We’re very excited about what’s co
 
 ### **RESOURCES:**
 
-- Introduction: https://discuss.linuxcontainers.org/t/introducing-microcloud/15871
+- Documentation: https://canonical-microcloud.readthedocs-hosted.com/en/latest/
```

Comment:
Not a blocker just wondering why we don't serve this under e.g. `documentation.ubuntu.com/microcloud`?

Reply:
This would be great, but needs approval from @evildmp.